app/models/appointment.rb

```diff
@@ -17,4 +20,18 @@ def assign_to_guider
            .first
     self.guider = slot.guider if slot
   end
+
+  private
+
+  def not_within_two_business_days
+    return unless start_at.present?
```

Comment:
I think just `return unless start_at` would suffice here and below. Also can you put a line of whitespace as breathing room between these guard clauses and the rest of the body?